File: array/OperationsOnAllElements/step1.rb
# 配列 A の要素数 N と整数 K, 配列 A の各要素 A_1, A_2, ..., A_N が与えられるので、配列 A に K がいくつ含まれるか数えてください。

n, k = gets.chomp.split(" ").map(&:to_i)
array = []
n.times do
    array << gets.to_i
end

puts array.count(k)
    

File: Python/LinearSearch/LocationOfTheSpecifiedValue/step1.py
# 整数 n と、数列 a_1, ... , a_n と、整数 k が与えられます。

# a_1, ... , a_n のうち k は何個あるかを求めてください。

n = int(input())
numbers = list(map(int,input().split()))
k = int(input())

print(numbers.count(k))
